api/v1alpha1/managednotification_types.go

```diff
@@ -294,4 +293,4 @@ func (c *Conditions) SetCondition(new NotificationCondition) {
 		}
 	}
 	*c = append(*c, new)
-}
\ No newline at end of file
+}
```

Comment:
Thinking out loud here - perhaps a helper function could be added which returns a boolean flagging if a managed notification should have a resolution SL sent?

ie.
```
func (m ManagedNotification) ShouldSendResolved() bool {
   // logic for deciding whether a resolved SL should be sent goes here
}
```

That will help make the subsequent `ocm-agent` code a bit cleaner / readable because it leaves the responsibility/logic of "when should I send a SL?" up to the API implementation, rather than deciding on its own.